### DISCUSSION TITLE
Loading a custom trained checkpoint/state_dict for inference/testing

### DISCUSSION THREAD

**Discussion by christianjosef27** [yangheng95/PyABSA#316] [Q&A]
Hi, I trained the multilingual model on my custom dataset which showed good evaluation metrics (85% etc.).
However, I do not know how to load the created custom checkpoints to test the trained model for inference.
(I have the automatically created folders containing the checkpoints, created during Training).

Can anyone help me where to research or how to do it?

**Comment by gald89** [comment]
Assuming that you are training using the following [script](https://github.com/yangheng95/PyABSA/blob/v2/examples-v2/aspect_opinion_sentiment_category_extraction/multitask_train.py):

```
from pyabsa import ABSAInstruction
model_path = r"" # newly generated checkpoint folder path here
generator = ABSAInstruction.ABSAGenerator(model_path)
```
